src/core/Akka/Actor/CoordinatedShutdown.cs

```diff
@@ -297,8 +297,8 @@ public Task<Done> Run(string fromPhase = null)
             if (_runStarted.CompareAndSet(false, true))
             {
                 var debugEnabled = Log.IsDebugEnabled;
-                Func<List<string>, Task<Done>> loop = null;
-                loop = remainingPhases =>
+
+                Task<Done> Loop(List<string> remainingPhases)
```

Comment:
Local function optimization in `CoordinatedShutdown`, which should have minimal impact if any.